### DISCUSSION TITLE
Areas we need to revisit when we tackle Aggregates

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#437] [Q&A]
Dealing with identity management for aggregates. List here any MARC fields/mappings we will need to revisit once we have an approach for how many/which works we will create for aggregates.

**Comment by CECSpecialistI** [comment]
MARC 505

**Comment by CECSpecialistI** [comment]
008 24-27

**Comment by pan-zhuo** [comment]
380 - Form of Work

**Comment by CECSpecialistI** [comment]
544 1st indicator 0 - Location of other archival materials note - Associated materials 
[MARC bibliographic](https://www.loc.gov/marc/bibliographic/bd544.html)
[Spreadsheet](https://docs.google.com/spreadsheets/d/1ZXiTvHvFkTQ9jshdh_jdp2-PXRdok9ya7OF9LuUbFFE/edit?usp=sharing)

**Comment by CECSpecialistI** [comment]
008/29 conference publications

**Comment by tmqdeborah** [comment]
I have put the PowerPoint from last weeks meeting <Aggregates in RDA Linked Data UW-MARC2RDA Presentation.20230806.pptx> in our Google Drive at:
https://docs.google.com/presentation/d/178yNNSwSYSI1KiLmKZr2-AgZG84F6-Bt/edit?usp=drive_link&ouid=109763205474483004874&rtpof=true&sd=true

I have added a few additional comments in the Notes for the slides with the green backgrounds and 3 additional slides at the end:

1. Unanswered questions
2. Ideas about identifying aggregate manifestations (AM)
3. Ideas about quick-and-dirty transformations

Deborah

**Comment by CECSpecialistI** [comment]
Sita sent along a copy of Damian Iseminger's slides from a talk he gave about aggregates at the PCC OpCo meeting in 2019. The file was too big for GitHub, so I put the slides in Google Drive  [here.](https://drive.google.com/file/d/1mbghSfZGXtJBqtWBf6gkDZViJzUfFzsW/view?usp=drive_link). Thank you for passing these along, @SitaKB !

**Comment by tmqdeborah** [comment]
With many re-starts, I am slowly working my way through a copy of every 10th record of approx. 5 million records from the LC database (as of Jan 2022) and identifying three categories of records:
1.	Aggregates: as far as I can tell, the pattern match logic (reasonably) identifies descriptions of aggregates that are primarily one of the following:
a.	Collection aggregates (might also be augmentation and/or parallel)
b.	Parallel aggregates (might also be augmentation)
c.	Augmentation aggregates of single expressions
2.	Possible aggregates: the pattern match logic needs further analysis
3.	Not aggregates: as far as I can tell, the pattern match logic (reasonably) identifies descriptions of single expressions without supplementary content.

To do this processing, I am using MARC Report software (now freely available for use at no charge and downloadable [here](https://marcreport.com/))

Using this software, I am running pattern matches that produce:
1.	a text file (.txt) displaying all MARC fields of records that matched a pattern (AND, OR, NOT, NONE, or NOWHERE) or series of patterns
2.	a MARC file (.mrc) of the records that matched the pattern
3.	a MARC file of the records that did not match the pattern; this file was then used for the next pattern match in my sequence of pulls.

I am still working on how to organize and present my findings to you, but in the meantime, I have put some sample result files into an Aggregates folder in the shared folder at: https://drive.google.com/drive/folders/15eAMtPfuEozpKp5RHpHKKwOSLLw9qN2O
In that folder, you will also find:
•	[FindingAggregates.20231023.xlsx](https://docs.google.com/spreadsheets/d/1PZ0eqVCkDNXtViPhmIR-auE3yp3rOqGB/edit#gid=1847438730): a spreadsheet that outlines the pattern matches I have used (so far), and some of the others that I plan to use. 
•	[Lists of terms for pattern matching](file:///D:/1.LCEnglish-Jan2022/Aggregates_2/%E2%80%94chosen%20from%20terms%20found%20in%20130%20or%20240%20fields%20in%20records%20in%20the%20LC%20database%20from%20Jan%202022):
1.	LCGFT: LCGFT Terms that might identify aggregates—chosen from https://www.loc.gov/aba/publications/FreeLCGFT/GENRE.pdf

2.	Non Music CCT: Non Music Conventional Collective Titles that identify aggregates—chosen from terms found in 130 or 240 fields in records in the LC database from Jan 2022

3.	Music CCT: Music Conventional Collective Titles that identify aggregates—chosen from terms found in 130 or 240 fields in records in the LC database from Jan 2022

Please add appropriate terms that you think might help to identify aggregates. Add a comment if you question any of my choices.

I am sharing this material to give you an idea of what I hope to present in a much more coherent form to, hopefully, assist us in discussion how we can identify and transform appropriate MARC records into descriptions of aggregate manifestations and the appropriate expressions/works that they embody.
